Resources/views/standard_layout.html.twig

```diff
@@ -132,10 +132,10 @@ file that was distributed with this source code.
                     {% spaceless %}
                         <a class="logo" href="{{ path('sonata_admin_dashboard') }}">
                             {% if 'single_image' == sonata_admin.adminPool.getOption('title_mode') or 'both' == sonata_admin.adminPool.getOption('title_mode') %}
-                                <img src="{{ asset(sonata_admin.adminPool.titlelogo) }}" alt="{{ sonata_admin.adminPool.title }}">
+                                <img src="{{ asset(sonata_admin.adminPool.titlelogo) }}" alt="{{ sonata_admin.adminPool.title|trans }}">
                             {% endif %}
                             {% if 'single_text' == sonata_admin.adminPool.getOption('title_mode') or 'both' == sonata_admin.adminPool.getOption('title_mode') %}
-                                <span>{{ sonata_admin.adminPool.title }}</span>
+                                <span>{{ sonata_admin.adminPool.title|trans }}</span>
```

Comment:
This will always call the translator, also there is no translation available. 

Would prefer a new config key where you could set the translation_domain.